source/android/adaptivecards/src/main/java/io/adaptivecards/renderer/readonly/ColumnRenderer.java

```diff
@@ -156,30 +157,32 @@ else if ((column = Column.dynamic_cast(baseCardElement)) == null)
         }
         else
         {
-            if (TextUtils.isEmpty(columnSize) || columnSize.equals(g_columnSizeStretch))
-            {
-                layoutParams = new LinearLayout.LayoutParams(LinearLayout.LayoutParams.MATCH_PARENT, LinearLayout.LayoutParams.MATCH_PARENT, 1);
-                returnedView.setLayoutParams(layoutParams);
-            }
-            else if (columnSize.equals(g_columnSizeAuto))
+            try
             {
-                layoutParams = new LinearLayout.LayoutParams(LinearLayout.LayoutParams.WRAP_CONTENT, LinearLayout.LayoutParams.MATCH_PARENT);
+                // I'm not sure what's going on here
```

Comment:
> // I'm not sure what's going on here [](start = 15, length = 37)

Hmm this comment makes me feel uneasy.. What is going on with the comment here?!